mailbox.h

```diff
@@ -107,6 +107,7 @@ struct Mailbox
   bool changed : 1;   /**< mailbox has been modified */
   bool readonly : 1;  /**< don't allow changes to the mailbox */
   bool quiet : 1;     /**< inhibit status messages? */
+  bool first_check_stats_done : 1; /**< true when the check have been done at least on time */
```

Comment:
Do we need a new bool?
Could we just check the `M->stats_last_checked` timestamp?

Reply:
OK, fair enough.